### PR TITLE
Don't fail the entire build if we fail to publish to python-nameless

### PR DESCRIPTION
**Pull request by dHannasch**
Not sure if this will actually work, my scripting is weak. But in theory this should provide a workaround until we can figure out why openssl is failing.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -47,10 +47,12 @@ install:
 script:
   - git config --global user.email "you@example.com"
   - git config --global user.name "Your Name"
+  - { # try
```
Comment (ionelmc) [inline .travis.yml]: Looks like that ain't valid yaml. Maybe replace with 
```
- | 
  { 
    openssl blablabla
  } || echo failed blabla
``` 

?

**Comment by ionelmc** [comment]
It's just a yaml list, so commands run individually. "{" has special meaning in yaml (start of hash or something like that - you can have vanilla json inside yaml)

**Comment by ionelmc** [comment]
Anyway, I've added you as a collaborator, that should allow the publishing to work on your prs I hope. Also please check if you can cancel appveyor builds (both here and https://ci.appveyor.com/project/ionelmc/python-nameless) - don't want to overload the appveyor queue because you know why :)

**Comment by dHannasch** [comment]
> please check if you can cancel appveyor builds (both here and https://ci.appveyor.com/project/ionelmc/python-nameless) - don't want to overload the appveyor queue

I'm currently looking at https://ci.appveyor.com/project/ionelmc/cookiecutter-pylibrary looking for a cancel button. (Not because I actually want to cancel the current build, just looking for it.)
AppVeyor's documentation doesn't exactly help, but I think there's supposed to be a "Cancel" button just to the left of where it says "Queued" in the right column for each job. https://help.appveyor.com/discussions/problems/2080-when-an-user-doesnt-have-permissions-to-cancel-build-it-should-show-an-error-message
I don't see any such button.
appveyor

I found an AppVeyor question, don't know if it's relevant or not. https://help.appveyor.com/discussions/problems/6493-user-permissions-button-to-cancel-build-not-visible

**Comment by ionelmc** [comment]
So i think i fixed the permissions now, take a look (I hope you used your .edu email for appveyor account, if not tell me which one).

**Comment by dHannasch** [comment]
I logged in to AppVeyor with my GitHub account. So...it would be the same email as in my commits? I think. I don't know how AppVeyor works. I do know that I'm still not seeing a cancel option anywhere.

**Comment by ionelmc** [comment]
image
Typo on my part or maybe you need to check spam folder?

**Comment by ionelmc** [comment]
This is how it should look:
image

**Comment by dHannasch** [comment]
Can you try David.A.Hannasch@gmail.com?

**Pull request merged**